internal/codegen/transform.go

```diff
@@ -0,0 +1,52 @@
+package codegen
+
+import (
+	"strings"
+	"unicode"
+)
+
+func toConfigCase(input string) string {
+	if input == "AWS SNS" {
+		return "Awssns"
+	} else if input == "3dEye" {
+		return "3DEye"
+	}
```

Comment:
These are the exception cases I mentioned in the previous comment. For "AWS SNS", it's supposed to be "AwsSns", but for some reason it's "Awssns" here.